pkg/util/utils.go

```diff
@@ -41,6 +41,26 @@ func WriteResponse(w http.ResponseWriter, code int, object interface{}) {
 	fmt.Fprintf(w, string(data))
 }
 
+// WriteErrorResponse will serialize 'object' to the HTTP ResponseWriter
+// with JSON formatted error response
+// using the 'code' as the HTTP status code
+func WriteErrorResponse(w http.ResponseWriter, code int, err error) {
+	type e struct {
+		Error string
+	}
+	data, err := json.Marshal(&e{
```

Comment:
How about just calling:
WriteResponse(w, code, e)
here and getting rid of the duplicate code?